scripts/generate_report.py

```diff
@@ -0,0 +1,163 @@
+
+#gr time random 2/1/18 5/1/18
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
```

Comment:
3rd time engine is called, this is super unnecessary please read up on [depndancy injection](https://www.google.com/webhp?sourceid=chrome-instant&rlz=1C1CHBF_enUS722US722&ion=1&espv=2&ie=UTF-8#q=dependency+injection&*)